src/languages/php.js

```diff
@@ -56,14 +56,19 @@ export default function(hljs) {
     end: /[ \t]*(\w+)\b/,
     contains: hljs.QUOTE_STRING_MODE.contains.concat(SUBST),
   });
+  const NOWDOC = hljs.END_SAME_AS_BEGIN({
+    begin: /<<<[ \t']*(\w+)'?\n/,
+    end: /[ \t]*(\w+)\b/,
+  });
```

Comment:
Should this not also support normal escapes and variable substitution, same as HEREDOC?